app/views/assignments/show.html.erb

```diff
@@ -32,206 +32,201 @@
       <%= render partial: 'read', locals:{ peer_review: false } %>
     </div>
   </div>
+  <% if @assignment.group_max > 1 %>
+    <div class='pane block'>
+      <h2><%= t('groups.student_interface.group_information') %></h2>
+      <div class='block-content'>
+        <% # If student has no group, inform them
+           if @grouping.nil? %>
+          <p class='notice'>
+            <% if @assignment.submission_rule.type == GracePeriodSubmissionRule && @assignment.group_max > 1%>
```

Comment:
This isn't the correct change. I think you meant to add this check for the other place I commented on above.